Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         // .package(url: /* package url */, from: "1.0.0"),
         .package(name: "danger-swift", url: "https://github.com/danger/swift.git", from: "3.0.0"),
         .package(name: "DangerSwiftHammer", url: "https://github.com/el-hoshino/DangerSwiftHammer", from: "0.2.0"),
-        .package(name: "DangerSwiftShoki", url: "git@github.com:yumemi/danger-swift-shoki.git", from: "0.1.0"),
+        .package(name: "DangerSwiftShoki", url: "git@github.com:yumemi/danger-swift-shoki.git", .branch("feature/ResponsibilityRefactor")),
```

Comment:
0.2.0 is not merged & released yet, so temporarily using branch specification in this PR as WIP. This will be fixed when  0.2.0 released